### ISSUE TITLE
New Contributors- Welcome to Crowd2Map :)

### ISSUE THREAD

**Issue by janetchapman** [janetchapman/contribute-Crowd2Map#1]
Hello there!
Thanks for showing interest in the project, we're so glad to have you on board. :)
Feel free to say hi and tell us more about yourself and/or how you relate to the project.

A little about me: I'm Anubha Maneshwar, a computer engineering student from Nagpur University, India. I am currently involved with Mozilla's Open Leadership Training, trying to carry forward the mission of an open and inclusive web.

This project is to create an online and offline platform for students who are lacking technical skills, to connect them with mentors and to make them aware about different opportunities available in the outside world! :')

Once again, welcome on board! :D

**Comment by farabi121** [comment]
Hi, I'm Salsabil from Bangladesh.

**Comment by janetchapman** [comment]
Thank you Sophie!  Did you look at our github?  What did you think? Perhaps we could chat at the end of the call?  Thanks Janet

Sent from Windows Mail

**Comment by mssososophie** [comment]
Yes let's have a call after, I'm ms.sophie.ashcroft on skype or ms.sophie.ashcroft@gmail.com on googlehangouts. I've had a look at the github but it's not clear where I can find more info/code on the repo.
I'll leave my gmail open and my skype on so please call me at your convenience.
